who_are_we.md

```diff
@@ -0,0 +1,111 @@
+
+Moshe suggested that Amitai take ownership of OG, to which he immediately replied "no." It was a huge responsibility and he felt completely unworthy.  Thirty minutes later, he said "yes, but you should stick around".  
+The next day Amitai already suggested to Moshe a complete re-write of Organic groups.
+
+They returned back to Israel, with commit access to one of the most important Drupal modules. Even back then, they knew this was an important milestone.
+
+
+## Theatrical Recognition
+
+Even though we were handed a significant module, Gizra was still quite unknown by the international community. We did however hire our first employees: an office manager who was also a project manager, and followed by a developer.
+
+Amitai, being the maintainer of Organic groups, indeed started [re-writing](https://www.drupal.org/node/567840) the module and upgrading its architecture to make it ready for the, back then, new Drupal 7 release.
+
+A year passed, Brice had a new baby girl, and Amitai as well, and it was time for DrupalCon Copenhagen (2010). Amitai submitted a session that was accepted - about the new OG for Drupal 7.
+
+Amitai decided to take a different approach to the talk. Rather than making a boring presentation about the nitty gritty technical aspects, what checkbox or button needs to be clicked, they gave a more theatrical session. Brice and Amitai performed in the first fifteen minutes of the session a completely [imaginary scenario](http://www.gizra.com/content/og7-session-drupalcon-copenhagen/) where Ban Ki Moon, the secretary general of the UN, approaches Gizra and commissions them with building sites for all the countries in the world. Through this story they illustrated the new features of OG.
+
+It's worth mentioning that as much as the scenario was completely ridiculous, in the end of the session one of the people in the audience approached Brice and Amitai and told them he was working in the UN, building an OG site for them - and in fact, their story wasn't so far from reality.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Amitaibu sets a pretty high standard for theatrical presentations.... history in the making! <a href="https://twitter.com/hashtag/drupalcon?src=hash">#drupalcon</a></p>&mdash; Gábor Hojtsy (@gaborhojtsy) <a href="https://twitter.com/gaborhojtsy/status/22181237352">August 26, 2010</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
once deployed it would look like a tweet

https://www.thegizraway.com/v/702cfd3d5f3b4627110fd82ba753e24a4fd37131/who_are_we.html

(you can see how any revision will look, via for example - https://www.gitbook.com/book/amitaibu/the-gizra-way/build/58244c551781491000d7076b)